src/styles.scss

```diff
@@ -2,3 +2,9 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer utilities {
+  .w-400px {
+    width: 400px;
+  }
+}
```

Comment:
You don't need this. There's better documented way of creating custom classes in Tailwind.